lib/money/money.rb

```diff
@@ -39,6 +39,13 @@ def from_subunits(subunits, currency_iso)
       new(value, currency)
     end
 
+    def rational(money1, money2)
+      money1.send(:arithmetic, money2) do
```

Comment:
Just a question, why send instead of a normal call?

Reply:
`send` allows us to access that private method. The alternative would be to make that method public which we don't really want or to make rational an instance method `money1.rational(money2)` (i think it's cleaner as a class method since it resembles more the `Rational(1,2)` syntax)